documentation/ntfs_to_gtfs_specs.md

```diff
@@ -12,6 +12,7 @@ The resulting GTFS feed is composed of the following objects:
 * [trips](#tripstxt)
 * [stop_times](#stop_timestxt)
 * [calendar_dates](#calendar_datestxt): only this file is provided instead of the calendar.txt file.
+* [attributions](#attributionstxt): only this file is provided instead of the calendar.txt file.
```

Comment:
Abused copy/paste : "only this file is provided instead of the calendar.txt file."